uktrade/data-hub-frontend
# Reactify OMIS assignee time page

## Description of change

The page for editing the estimated working hours for OMIS assignees has been reactified.

## Test instructions

Go to an OMIS order with assignees. You should be able to edit the estimated hours for the assignees.

Go to an OMIS order without assignees. You should see a message saying that you need to add assignees.

## Screenshots

### Before

<img width="746" alt="Screenshot 2023-10-31 at 14 01 46" src="https://github.com/uktrade/data-hub-frontend/assets/36161814/65643b9b-6ae7-4ca5-be84-116f61409a24">

### After

<img width="1000" alt="Screenshot 2023-10-31 at 14 01 18" src="https://github.com/uktrade/data-hub-frontend/assets/36161814/dea73279-7466-4375-9f01-acdfdee43fec">


## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
